addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1165,8 +1165,9 @@ def _send_transient_message(self, partner_to, content):
         """
         self.env['bus.bus']._sendone(partner_to, 'discuss.channel/transient_message', {
             'body': f"<span class='o_mail_notification'>{content}</span>",
-            'model': self._name,
-            'res_id': self.id,
+            'model': self._name,  # deprecated
+            'res_id': self.id,  # deprecated
+            'originThread': {'model': self._name, 'id': self.id},
```

Comment:
It's really sad to have to send the data twice, this will generate a lot of network for no reason.
Ideally the PR should work without changing formatters.